build.gradle

```diff
@@ -13,8 +13,8 @@ repositories {
 }
 
 compileGroovy {
-    sourceCompatibility = '1.6'
-    targetCompatibility = '1.6'
+    sourceCompatibility = '1.8'
+    targetCompatibility = '1.8'
```

Comment:
This isn't strictly needed but should be fine and it makes the java code tidier

Reply:
true, I'd keep as it is though due to the usage of our older Android SDK that runs on Java 6/7, removing the lambdas should be already enough.
We'd go fully with Kotlin in the new Plugin version, targeting Java 8 of course.

Reply:
Fair enough, though this only affects the build scripts and nothing else.

I'll revert 👍

Btw, I reached out on your personal Gmail address regarding the new plugin. Did you receive it or did it go to spam?

Reply:
hey, yep, I've gotten it and forwarded it to my Sentry email but I didn't have the time just yet, gonna do it within the next days, sorry about that, just wanna give more like a concrete direction instead of a short reply :)

Reply:
Ah, take your time! I just wanted to make sure you received it.